### PR TITLE
task: add page

### PR DESCRIPTION
**Pull request by DavidLambauer**
----

- [ x ] The page (if new), does not already exist in the repo.

- [ x ] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [ x ] The page has 8 or fewer examples.

- [ x ] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [ x ] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: pages/common/taskwarrior.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# taskwarrior
+
+> An open source todo application for your terminal.
+
+- Add a Task:
```
Comment (sbrl) [inline pages/common/taskwarrior.md]: Perhaps `Task` should be in lowercase?

File: pages/common/taskwarrior.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+- List all tasks:
+
+`task list`
+
+- Mask a task as done:
+
+`task {{task_id}} done`
+
+- Add a project to a task:
```
Comment (sbrl) [inline pages/common/taskwarrior.md]: Perhaps `project to a task` would read better as `task to a project`?

**Comment by DavidLambauer** [comment]
Hi, thank you for the comments, you are totally right!

**Comment by agnivade** [comment]
We already have a page for this. See "pages/common/task.md"

**Comment by DavidLambauer** [comment]
You're right! Sorry I didn't see that page. I will go over it and update it if necessary. Therefore we can close this PR.

**Pull request closed without merging**